src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int SIZE_OF_THE_ARRAY = 10;
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
The FigureSupplier instance should be created using dependency injection or at least its constructor. This would make the code more flexible and easier to test.

Reply:
done